csharp/src/Ice/Property.cs

```diff
@@ -4,9 +4,10 @@ namespace IceInternal;
 
 public sealed class Property
 {
-    public Property(string pattern, bool deprecated, string deprecatedBy)
+    public Property(string pattern, string defaultValue, bool deprecated, string deprecatedBy)
```

Comment:
What about using a readonly record class?

Reply:
Let's do that in a follow-up PR.